api/v1alpha1/virtualmachine_types.go

```diff
@@ -13,8 +13,9 @@ type VirtualMachinePowerState string
 
 // See govmomi.vim25.types.VirtualMachinePowerState
 const (
-	VirtualMachinePoweredOff VirtualMachinePowerState = "poweredOff"
-	VirtualMachinePoweredOn  VirtualMachinePowerState = "poweredOn"
+	VirtualMachinePoweredOff VirtualMachinePowerState    = "poweredOff"
+	VirtualMachinePoweredOn  VirtualMachinePowerState    = "poweredOn"
+	VirtualMachineReady      VirtualMachineConditionType = "Ready"
```

Comment:
This doesn't belong here. Start a new const for the VirtualMachineConditionType's, that's below the type definition like is done elsewhere in this file.